backend/apps/api_auth/views.py

```
@@ -0,0 +1,153 @@
+        # 解析请求数据
+        data = json.loads(request.body)
+        username = data.get('username', '').strip()
+        password = data.get('password', '').strip()
+
+        # 从环境变量读取配置的账号密码
+        default_username = os.environ.get('DEFAULT_USERNAME', 'admin')
+        default_password = os.environ.get('DEFAULT_PASSWORD', 'admin123')
+
+        # 验证账号密码
+        if username == default_username and password == default_password:
+            # 生成 token（使用 secrets 生成安全的随机 token）
```

Ruff (0.14.13)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)